DEVDOCS-5033: [new] B2B getting started + GraphQL Storefront API auth narrative (#288)
bigcommerce/docs: docs/api-docs/storefront/graphql/graphql-carts-and-checkout.mdx

No additional scopes are required to use the GraphQL Storefront API.

To generate a bearer token your application can pass to authenticate calls to the GraphQL Storefront API, use either the [Create a storefront token](/docs/rest-authentication/tokens#create-a-token) or the [Create a customer impersonation token](/docs/rest-authentication/tokens/customer-impersonation-token#create-a-token) REST endpoint. On a Stencil storefront, you can also access a token at render time using a Handlebars property. Learn more about [Using auto-generated tokens in Stencil themes](/docs/start/authentication/graphql-storefront#using-auto-generated-tokens-in-stencil-themes).

> For more information, see [Creating a token](/docs/start/authentication/graphql-storefront#creating-a-token) in Authenticating requests to the GraphQL Storefront API and [Dynamic Tokens](/docs/start/authentication#bigcommerce-generated-jwts) in the Authentication and example requests article.

Author: ```suggestion
For more information, see [Creating a token](/docs/start/authentication/graphql-storefront#creating-a-token) in Authenticating requests to the GraphQL Storefront API and [dynamic tokens](/docs/start/authentication#bigcommerce-generated-jwts) in the Authentication and example requests article.
```

No additional scopes are required to use the GraphQL Storefront API.

> To generate a bearer token your application can pass to authenticate calls to the GraphQL Storefront API, use either the [Create a storefront token](/docs/rest-authentication/tokens#create-a-token) or the [Create a customer impersonation token](/docs/rest-authentication/tokens/customer-impersonation-token#create-a-token) REST endpoint. On a Stencil storefront, you can also access a token at render time using a Handlebars property. Learn more about [Using auto-generated tokens in Stencil themes](/docs/start/authentication/graphql-storefront#using-auto-generated-tokens-in-stencil-themes).

Author: ```suggestion
To generate a bearer token to authenticate calls to the GraphQL Storefront API, use either the [Create a storefront token](/docs/rest-authentication/tokens#create-a-token) or the [Create a customer impersonation token](/docs/rest-authentication/tokens/customer-impersonation-token#create-a-token) REST endpoint. On a Stencil storefront, you can also access a token at render time using a Handlebars property. Learn more about [Authenticating requests to the GraphQL Storefront API](/docs/start/authentication/graphql-storefront).
```

### Articles

* [GraphQL Storefront API Overview](/docs/storefront/graphql)

> * [GraphQL Storefront API Overview: Using auto-generated tokens in Stencil themes](/docs/start/authentication/graphql-storefront#using-auto-generated-tokens-in-stencil-themes)

Author: ```suggestion
* [Authenticating requests to the GraphQL Storefront API](/docs/start/authentication/graphql-storefront)
```
